Library/Homebrew/test/sbom_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "sbom"
+
+RSpec.describe SBOM, :needs_network do
```

Comment:
What does it use the network for here?

Reply:
Fetching the schema to validate against

Reply:
@SMillerDev ok, cool. If it's not large: might make sense to vendor it. If it is: fine, np.

Reply:
I'm not really sure how to vendor it, so I'd prefer to keep it like this for now.

Reply:
Fine with me!